libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/FormatUtils.java

```diff
@@ -32,4 +32,27 @@ public static final String formatInt(int value) {
     public static final String formatDecimal(int value) {
         return DECIMAL_INSTANCE.get().format(value).toString();
     }
+
+    /*
+     * returns the passed long formatted has an human readable filesize. Ex: 10 GB
+     */
+    public static final String formatFileSize(long size) {
+        if (size <= 0) {
+            return "0";
+        }
+        final String[] units = new String[] {"B", "kB", "MB", "GB", "TB" };
```

Comment:
I'd defer to @akirk to make sure the i18n approves this way of showing measuring units, which might make sense in English but might not be the case in other languages. Adding the `[Needs i18n Review]` tag.